src/marqo/tensor_search/api.py

```diff
@@ -187,6 +188,12 @@ def memory():
     return memory_profiler.get_memory_profile()
 
 
+@app.get('/validation')
```

Comment:
Would it be better if we use `POST` and put the settings_object in the request body?